tests/input/test_input_gg_checks.py

```diff
@@ -0,0 +1,81 @@
+"""Test suite for parse_config function."""
+
+import pytest
+from gatorgrade.input.in_file_path import parse_config
+
+
+def test_parse_config_gg_check_in_file_context_contains_file():
+    """Test to make sure gatorgrader checks that have a file context include the file name in the list of lists."""
+    # Given the path to the test yml file
+    config = "tests/input/yml_test_files/gatorgrade_one_gg_check_in_file_context.yml"
+    # when the parse_config function is run
+    output = parse_config(config)
+    # then assert that the file path exists in the first index of the list
+    assert "file.py" in output["gatorgrader"]
```

Comment:
Since `output["gatorgrader"]` will be a list of lists, the author of this test probably meant to access the first index of this value--`output["gatorgrader"][0]`--as is used in `test_parse_config_setup_shell_checks`.

Reply:
Yes! Thank you for this point!